docs/TuxActionStores.md

```diff
@@ -0,0 +1,170 @@
+`ActionStores` are an implementation on top of the basic `TuxStore`. Thus, they have all of the same properties. The only difference is that `ActionStores` expect to have a `register` method passed in. If this `register` key is defined, then the ActionStore will invoke the method with the store as the context and use the object the method returns to register the store with `TuxActions`. Let's take a look at an example
+
+```
+    var actionStore = ActionStores.createStore({
+      _messages: [],
+
+      onCreate: function (message) {
+        this._messages.push(message);
+        this.emitChange();
+      },
+
+      register: function () {
+        return {
+          messages: {
+            create: this.onCreate
+          }
+        };
+      }
+    });
+```
+
+Below is the API documentation for this method.
+### <a id="ActionStores-createActionStore"></a>ActionStores.createStore [#](#ActionStores-createStore)
+Returns an `actionStore` instance based on the passed in `storeProps`.
+
+```
+    var actionStore = ActionStores.createStore(storeProps);
+```
+
+#### <a id="storeProps"></a>Parameter - `storeProps` - type: OBJECT - required [#](#storeProps)
+StoreProps is an object containing actions that will encapsulate the store's data, with an optional register method which, if provided, will automatically register all the actions with the newly created store. Let's see what this looks like:
+
+```
+    var messageStoreProperties = {
+      _messages: [],
+
+      onCreate: function (message) {
+        this._messages.push(message);
+        this.emitChange();
+      },
+
+      register: function () {
+        return {
+          messages: {
+            create: this.onCreate
+          }
+        };
+      }
+    };
+
+```
+
+#### <a id="storeProps-register"></a>Parameter Method - `storeProps.register` - type: FUNCTION - optional [#](#createActionStore)
+The `register` method is an optional key on the `storeProps` parameter which, if defined, will invoke `Actions.register`, passing in the store as the first argument and the object returned by this `storeProps.register` method as the second argument. Here is what the `register` method performs on the backend:
+
+```
+    Actions.register(actionStore, actionStore.register.call(actionStore));
+```
+
+ Now lets look at an example of `storeProps.register`:
+
+```
+    storeProps.register = function () {
+      return {
+        messages: {
+          create: this.onCreate;
+        }
+      };
+    };
+```
+Note that `this` in the above example refers to the store being created.
+
+Furthermore, this syntax abstracts away even more complexity by allowing you to register listeners with one or multiple actions and one or multiple categories simultaneously.
+
+```
+    {
+      CATEGORY1: {
+        CATEGORY1_ACTION1: this.ONTHISACTION,
+        CATEGORY1_ACTION2: this.ONANOTHERACTION,
+      },
+      CATEGORY2: {
+        ACTION1: this.ONFINALACTION
+      }
+    }
+```
+
```

Comment:
Add a section with what the `createStore` method returns:

#### <a id="createStore-actionStore"></a>Return - `actionStore` - OBJECT [#](#createStore-actionStore)
Returns a `TuxStore` instance.